code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -268,16 +268,17 @@
 
 /obj/item/rig_module/modular_injector/medical
 	name = "mounted medical injector"
-	desc = "A specialized system for inserting chemicals to pacients"
+	desc = "A specialized system for injecting chemicals in patients."
 	price_tag = 3750
 	max_injection_amount = 60
 	max_beakers = 6
 	usable = 0
 	selectable = 1
 	disruptive = 1
 	initial_beakers = list(
+		list(/obj/item/reagent_containers/glass/beaker/large, "dexalinp",60),
+		list(/obj/item/reagent_containers/glass/beaker/large, "tramadol",60),
 		list(/obj/item/reagent_containers/glass/beaker/large, "bicaridine", 60),
-		list(/obj/item/reagent_containers/glass/beaker/large, "inaprovaline",60),
```

Comment:
Please do not remove inaprovaline from this

Reply:
After livermed Inaprovaline has become, once more, completely useless as a chem to stabilize a patient, but sure.